src/Logger/BasicLogger.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Mollie\Logger;
+
+use Exception;
+use PrestaShopLogger;
+
```

Comment:
can we not create out own loggers and rely on PSR logger interface ? https://www.php-fig.org/psr/psr-3/